### PR TITLE
Introduce exists method and support expiration for add methods

### PR DESCRIPTION
**Pull request by rterentiev**

### CODE REVIEW

File: src/main/java/org/springframework/data/aerospike/core/AerospikeTemplate.java [hunk 1]
```diff
@@ -268,6 +268,21 @@ public void scanCallback(Key key, Record record)
 	}
 
 	@Override
+	public boolean exists(Serializable id, Class<?> type) {
+		Assert.notNull(id, "Id must not be null!");
```
Comment (Aloren) [inline src/main/java/org/springframework/data/aerospike/core/AerospikeTemplate.java]: type can not be null as well. add assert pls

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateTests.java [hunk 2]
```diff
@@ -816,4 +816,13 @@ public void shouldTranslateException() {
 		});
 	}
 
+	@Test
+	public void shouldExistsAfterInsert() {
```
Comment (Aloren) [inline src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateTests.java]: exists_shouldReturnTrueIfValueIsPresent
exists_shouldReturnFalseIfValueIsAbsent

**Comment by Aloren** [comment]
Please also replace exists method in `SimpleAerospikeRepository` to use `exists` instead of `findOne`

File: src/main/java/org/springframework/data/aerospike/core/AerospikeTemplate.java (hunk 1 above)
Comment (rterentiev) [inline src/main/java/org/springframework/data/aerospike/core/AerospikeTemplate.java]: Added asserts for ```exists``` and both ```add``` method.

**Pull request merged**